unity-client/Assets/Scripts/MainScripts/Debugging/WSS/WSSController.cs

```diff
@@ -302,11 +302,17 @@ void Update()
                             case "RemoveUserProfilesFromCatalog":
                                 UserProfileController.i.RemoveUserProfilesFromCatalog(msg.payload);
                                 break;
+                            case "ActivateRendering":
+                                renderingController.ActivateRendering();
+                                break;
                             case "DeactivateRendering":
                                 renderingController.DeactivateRendering();
                                 break;
-                            case "ActivateRendering":
-                                renderingController.ActivateRendering();
+                            case "ReportFocusOn":
+                                CommonScriptableObjects.focusState.Set(true);
```

Comment:
As a rule of thumb I try to avoid setting values of a SO from different places. Cannot we use the Bridge here so we centralize the setter of this value there?

Reply:
makes sense, done!